main/src/library/FilePath.flix

```diff
@@ -155,3 +155,49 @@ pub eff FilePath {
     pub def list(f: String): List[String]
 
 }
+
+mod FilePath {
+
+    import java.io.File
+    import java.lang.Class
+    import java.lang.Exception
+    import java.lang.{String => JString}
+    import java.nio.charset.StandardCharsets
+    import java.nio.file.attribute.BasicFileAttributes
+    import java.nio.file.Files
+    import java.nio.file.LinkOption
+    import java.nio.file.Path
+    import java.nio.file.Paths
+
+    ///
+    /// Handles the `FilePath` effect of the given function `f`.
+    ///
+    /// In other words, re-interprets the `FilePath` effect using the `IO` effect.
+    ///
+    pub def handle(f: a -> b \ ef): a -> Result[IOError, b] \ (ef - FilePath) + IO = x ->
+        try {
+            Ok(f(x))
+        } with FilePath {
+            def exists(f, k) = {
+                let existsRes = try {
+                    let path = Paths.get(f);
+                    let linkOpts: Vector[LinkOption] = Vector.empty();
+                    let exists = Files.exists(path, linkOpts);
```

Comment:
Can Flix not resolve  the call to exists without the Vector?

Reply:
It depends. I've found that excluding the varargs is flaky in general, partly because of https://github.com/flix/flix/issues/9067, but also because it will sometimes randomly decide that the function override doesn't exist when changing unrelated parts of the program.

So actually, we should probably add one more for the `Paths.get()` call.